library/Message/StandardRenderer.php

```diff
@@ -29,15 +31,17 @@ final class StandardRenderer implements Renderer
 
     private readonly Stringifier $stringifier;
 
-    public function __construct(?Stringifier $stringifier = null)
-    {
+    public function __construct(
+        ?Stringifier $stringifier = null,
```

Comment:
Gotta create our own stringifier to remove the Callable one, and also create one to parse a `Path` and `Name` value objects, that will create more flexibility. Maybe even `Trans` and `Raw` value objects could be a thing.